cmd/pool/account.go

```diff
@@ -119,10 +124,30 @@ func newAccount(ctx *cli.Context) error {
 
 	req := &poolrpc.InitAccountRequest{
 		AccountValue: amt,
-		Fees: &poolrpc.InitAccountRequest_ConfTarget{
+		Initiator:    defaultInitiator,
+	}
+
+	satPerVByte := ctx.Uint64("sat_per_vbyte")
+	confTarget := ctx.Uint64("conf_target")
+
+	switch {
+	case satPerVByte != 0:
+		// Enforce a minimum fee rate of 253 sat/kw by rounding up if 1
+		// sat/byte is used.
+		feeRate := chainfee.SatPerKVByte(satPerVByte * 1000).FeePerKWeight()
+		if feeRate < chainfee.FeePerKwFloor {
+			feeRate = chainfee.FeePerKwFloor
+		}
+		req.Fees = &poolrpc.InitAccountRequest_FeeRateSatPerKw{
+			FeeRateSatPerKw: uint64(feeRate),
+		}
+	case confTarget > 0:
```

Comment:
Because of the default value, the `confTarget` here will always be greater than 0 (unless set explicitly to 0 by the user). You need to use `ctx.IsSet("conf_target")` here.